projects/tomcat/Dockerfile

```diff
@@ -16,15 +16,16 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder-jvm
 
```

Comment:
I assume this won't be able to run on a machine running Java17 right? Installing java21 here won't install it on the machines the fuzzer actually runs on. https://github.com/google/clusterfuzz/blob/oss-fuzz/docker/base/Dockerfile#L91

Reply:
gcr.io/oss-fuzz-base/base-builder-jvm is based on Ubuntu 20.04 which uses OpenJDK 11 as the default JDK when you install the default-jdk package. The base OS is old but fortunately Canonical backported OpenJDK 21, so the PR simply replaces the default-jdk package with openjdk-21-jdk and updates the paths from 11 to 21.